### DISCUSSION TITLE
Sub-file Lookup in Note Search

### DISCUSSION THREAD

**Discussion by protean-onion** [TriliumNext/Trilium#3991] [General]
Hello!

I have many different files titled "Music". For example, many are under files about different films: "Good Will Hunting/Music". Some are under disciplines. For example, "Books/Music".

The note search doesn't allow me to specify the file under which the "Music" file _I_ want to open is. It only allows me to specify the filename I want to open, not the file under which it is located. It would be nice if I could type something like "Books/Music" to go to exactly that file.

Is this possible? Would it require too much programming?

Thanks for your time. I look forward to your response.

**Comment by meichthys** [comment]
The JumpTo feature does fuzzy search so you can search for "Books/Music" and the Music page under books should show as the top result.
To use the jumpTo feature just press `ctrl-j` or press the right arrow button in the sidebar:
![image](https://github.com/zadam/trilium/assets/10717998/35891d23-d074-477d-aaff-26530fb8cf6d)

**Comment by zadam** [comment]
besides what @meichthys already mentioned, it's also possible to define "ancestor" in full search:

![image](https://github.com/zadam/trilium/assets/617641/9a97574f-d866-4923-b9f0-e1458f6a132a)
